cypress/integration/admin/loggingInAndOut.js

```diff
@@ -20,8 +20,8 @@ describe("As an admin, I want to log in so that I can access the service.", () =
     GivenIAmAnAdmin();
     WhenIVisitTheLandingPage();
     AndIClickTheLinkToManageYourTrustPage();
-    ThenIVisitTheManageYourTrustLoginPage();
-
+    // ThenIVisitTheManageYourTrustLoginPage();
```

Comment:
I think you forgot to remove this comment?

Reply:
Yeah, forgot to remove. Just removed it